apis/fluentd/v1alpha1/tests/expected/fluentd-cluster-cfg-output-null.cfg

```
@@ -0,0 +1,30 @@
+<source>
+  @type  forward
+  bind  0.0.0.0
+  port  24224
+</source>
+<match **>
+  @id  main
+  @type  label_router
+  <route>
+    @label  @a2170d34e9940ec56d328100e375c43e
+    <match>
+      namespaces  default,kube-system
+    </match>
+  </route>
+</match>
+<label @a2170d34e9940ec56d328100e375c43e>
+  <filter **>
+    @id  ClusterFluentdConfig-cluster-fluentd-config::cluster::clusterfilter::fluentd-filter-0
+    @type  record_transformer
+    enable_ruby  true
+    <record>
+      kubernetes_ns  ${record["kubernetes"]["namespace_name"]}
+    </record>
+  </filter>
+  <match **>
+    @id  ClusterFluentdConfig-cluster-fluentd-config::cluster::clusteroutput::fluentd-output-null-0
+    @type  null
+    never_flush  false
+  </match>
+</label>
```

```suggestion
</label>

```

Using .editorconfig will ensure all files end with a whiteline. We have a `.editorconfig` configuration in place on the repo.

---

noted in vscode, we need to install the plugin to read the `.editorconfig` https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig